app/src/lib/stores/sign-in-store.ts

```diff
@@ -34,10 +34,10 @@ const EnterpriseTooOldMessage = `The GitHub Enterprise version does not support
  * store can be in save for the unitialized state (null).
  */
 export enum SignInStep {
-  EndpointEntry,
-  Authentication,
-  TwoFactorAuthentication,
-  Success,
+  EndpointEntry = 'EndpointEntry',
```

Comment:
Doing this because the numbers are not very readable when converted to string

Reply:
luv string enums 👍 💯